testsuite/MDAnalysisTests/analysis/test_base.py

```diff
@@ -197,6 +197,47 @@ def test_start_stop_step(u, run_kwargs, frames):
     assert_almost_equal(an.times, frames+1, decimal=4, err_msg=TIMES_ERR)
 
 
```

Comment:
It'd be too much to ask to add a test for every analysis method, but could we maybe throw in an extra test for RMSD that shows that passing frames works with that as expected?

Reply:
I added a test to test_rmsd.py